tracing-subscriber/src/fmt/time.rs

```diff
@@ -96,6 +96,61 @@ impl FormatTime for Uptime {
     }
 }
 
+/// Print the current UTC time using the [RFC 3339](https://tools.ietf.org/html/rfc3339)
+/// format (for example, 2019-10-16T03:16:56.992060752+00:00).
+///
+/// Note: RFC 3339 is a subset of [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)
+#[cfg(feature = "chrono")]
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Default)]
+pub struct Rfc3339;
+
+#[cfg(feature = "chrono")]
+impl FormatTime for Rfc3339 {
+    fn format_time(&self, w: &mut dyn fmt::Write) -> fmt::Result {
+        write!(w, "{} ", chrono::Utc::now().to_rfc3339())
+    }
+}
+
+/// Print the current time using the given format string.
+/// See [`chrono::format::strftime`](https://docs.rs/chrono/0.4.9/chrono/format/strftime/index.html)
+/// for details on the supported sytax.
+#[cfg(feature = "chrono")]
+#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+pub struct ChronoFmt {
+    fmt_str: &'static str,
+    utc: bool,
+}
+
+#[cfg(feature = "chrono")]
+impl ChronoFmt {
+    /// Use the given format string.
+    /// See [`chrono::format::strftime`](https://docs.rs/chrono/0.4.9/chrono/format/strftime/index.html)
+    /// for details on the supported sytax.
+    pub fn new(fmt_str: &'static str) -> Self {
+        ChronoFmt {
+            fmt_str,
+            utc: false,
+        }
+    }
+
+    /// Use UTC instead of local time
+    pub fn use_utc(mut self) -> Self {
+        self.utc = true;
+        self
+    }
+}
+
+#[cfg(feature = "chrono")]
+impl FormatTime for ChronoFmt {
+    fn format_time(&self, w: &mut dyn fmt::Write) -> fmt::Result {
+        if self.utc {
```

Comment:
I wonder if these should maybe just be separate types? TIOLI...

Reply:
`ChronoUtc` and `ChronoLocal`? Happy to split it up if you prefer